src/utils.ts

```diff
@@ -170,31 +250,32 @@ export const processRemovePositionEvent = (
 
   ownerData.closed.push({
     events: [correspondingEvent, event],
+    points: calculatePointsForClosedPosition(event),
   });
 
   eventsObject[ownerKey] = ownerData;
 };
 
-export const extractEvents = (
+export const extractEvents = async (
```

Comment:
It looks like the function in becoming a core for our calculations. It should be moved to main script file. Let's move it in next PR